src/libclang/libclang_parser.cpp

```diff
@@ -373,6 +375,28 @@ void libclang_compile_config::do_set_flags(cpp_standard standard, compile_flags
         else
             add_flag("-std=c++1z");
         break;
+    case cpp_standard::cpp_17:
+        if (flags & compile_flag::gnu_extensions)
+            add_flag("-std=gnu++17");
+        else
+            add_flag("-std=c++17");
+        break;
+    case cpp_standard::cpp_2A:
+        if (flags & compile_flag::gnu_extensions)
+            add_flag("-std=gnu++2A");
+        else
+            add_flag("-std=c++2A");
+        break;
+    case cpp_standard::cpp_20:
+        if (libclang_parser::libclang_minor_version() >= 6) {
```

Comment:
The minor version of libclang does not correspond to the clang version unfortunately. clang version 6 is libclang minor version 45.